src/VisualStudio/Core/Def/Implementation/Preview/PreviewUpdater.cs

```diff
@@ -35,14 +33,14 @@ public void CloseWorkspace()
             }
         }
 
-        public void UpdateView(Document document, SpanChange spanSource)
+        public void UpdateView(TextDocument document, SpanChange spanSource, bool isAdditionalDocument)
```

Comment:
is isAdditionalDocument need to be passed down? whoever needs that flag should be able to get that info from document passed down I think.